src/test/java/com/google/sps/unit/ServletUtilsTest.java

```diff
@@ -64,21 +64,21 @@ public void testHTTPRequestGetInvalidParameter() throws Exception {
     verify(printWriter).println("No anInvalidKey in the query URL.");
   }
 
-  // Test ServletUtils getRquestParam() function correctly returns empty Optional container when
+  // Test ServletUtils getRequestParam() function correctly returns empty Optional container when
   // provided parameter key is null.
   @Test
-  public void testHTTPRequestGetNullParameter() throws Exception {
+  public void testGetRequestParamWithNullParameterName() throws Exception {
```

Comment:
nice, I like the extra descriptions

Can you change it to the format `methodUnderTest_condition1_condition2_expectedOutput`? I don't care which format we use much, but we should keep the naming scheme consistent across our codebase :) 

Similar to https://github.com/googleinterns/step190-2020/blob/master/src/test/java/com/google/sps/integration/ElectionListIT.java